store/migrate.go

```diff
@@ -101,3 +102,27 @@ func migrateToV3(tx *sql.Tx) error {
 	}
 	return nil
 }
+
+func migrateToV4(tx *sql.Tx) error {
+	for _, t := range []string{
+		"CREATE TABLE aciinfo_tmp (blobkey string, name string, importtime time, lastusedtime time, latest bool);",
+		"INSERT INTO aciinfo_tmp (blobkey, name, importtime, latest) SELECT blobkey, name, importtime, latest from aciinfo",
```

Comment:
It returns local time. I'll keep it the way it is now but I'll add a comment about it.